test/src/code_field/text_editing_value_select_test.dart

```diff
@@ -0,0 +1,100 @@
+// ignore_for_file: avoid_redundant_argument_values
+// ignore_for_file: prefer_const_constructors
+
+import 'package:flutter/widgets.dart';
+import 'package:flutter_code_editor/flutter_code_editor.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('TextEditingValue ', () {
+    test('select', () {
+      final examples = [
+        //
```

Comment:
Is this empty comment deliberate?

Reply:
Yes, it preserves empty lines between examples.